### PR TITLE
update the hand config file

### PR DESCRIPTION
**Pull request by sunjiekcl**
Update the hand config file, test it in the vienna robot in the integration meeting.

### CODE REVIEW

File: robotino_bringup/launch/kclhand_launch_new.launch [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
```
Comment (NadiaHG) [inline robotino_bringup/launch/kclhand_launch_new.launch]: please rename this file to kclhand.launch

File: robotino_bringup/robots/kcl-hand/config/hand.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-# Vienna hand config parameters 
+# Innsbruck hand config parameters
```
Comment (NadiaHG) [inline robotino_bringup/robots/kcl-hand/config/hand.yaml]: is this the  config for the Innsbruck hand? if yes, please move the file to robotino_bringup/robots/uibk-robotino2/config/

File: robotino_bringup/robots/tuw-robotino2/config/hand.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-# Vienna hand config parameters 
+# KCL/Freiburg hand config parameters
```
Comment (NadiaHG) [inline robotino_bringup/robots/tuw-robotino2/config/hand.yaml]: is this the  config for the Freiburgs hand? if yes, please move the file to robotino_bringup/robots/alufr-robotino/config/

**Review by ipa-nhg — CHANGES_REQUESTED** [review]

File: robotino_bringup/launch/kclhand_launch_new.launch (hunk 1 above)
Comment (sunjiekcl) [reply robotino_bringup/launch/kclhand_launch_new.launch]: done. Remove the old version kclhand.launch file

File: robotino_bringup/robots/kcl-hand/config/hand.yaml (hunk 2 above)
Comment (sunjiekcl) [reply robotino_bringup/robots/kcl-hand/config/hand.yaml]: just wrong name. should be KCL hand. already fix.

File: robotino_bringup/robots/tuw-robotino2/config/hand.yaml (hunk 3 above)
Comment (sunjiekcl) [reply robotino_bringup/robots/tuw-robotino2/config/hand.yaml]: done. should be vienna hand config parameters. Already fix.

**Review by ipa-nhg — APPROVED** [review]
@sunjiekcl thanks a lot!

**Pull request merged**